### PR TITLE
Update copytext and fix incorrect links

### PR DESCRIPTION
**Pull request by nathakits**
Fix incorrect hash links
update outdated copytext and moving some content around

@willpatera - please review this PR.

### CODE REVIEW

File: developer-docs/windows.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ For discussion or questions on Windows installation head over to the [Pupil Goog
 
 **Visual C++ Runtime**
 
-* Install Visual Studio 2015 Community Update 3
+* Install Visual Studio Community 2017
```
Comment (willpatera) [inline developer-docs/windows.md]: Remove this change - I can not confirm that 2017 works just yet.

File: getting-started/capture-workflow.md [hunk 2]
```diff
@@ -77,10 +77,10 @@ The default `recordings` directory will have the following hierarchy.
 
 * `recordings`
   * 2016-04-05
-    * 001
-    * 002
-    * 003
-    * ####
+	    * 001
```
Comment (willpatera) [inline getting-started/capture-workflow.md]: Why so many spaces? Isn't `.md` syntax just using two whitespaces for indented lists.

File: getting-started/intro.md [hunk 3]
```diff
@@ -21,4 +21,4 @@ Once you have a Pupil Headset all you need to do is install the Pupil apps on a
   </p>
 </div>
 
-We are always working on new features, fixing bugs, and making improvements. Make sure to visit the release page frequently to download the latest version and follow the Pupil Labs [blog](https://pupil-labs.com/blog "Pupil Labs Blog) for updates. 
+We are always working on new features, fixing bugs, and making improvements. Make sure to visit the release page frequently to download the latest version and follow the Pupil Labs [blog](https://pupil-labs.com/blog) for updates.
```
Comment (willpatera) [inline getting-started/intro.md]: You can keep "Pupil Labs Blog" - but it just needs a closing `"` symbol. This serves as the `title` attr of the link in md I believe.

File: pupil-hardware/pupil-headset.md [hunk 4]
```diff
@@ -56,7 +56,7 @@ Depending on your configuration, some additional parts may be included with your
       <p style="padding: 10px;">You can <b>slide</b> the eye camera arm along the track.</p>
     </div>
     <div class="Grid-cell">
-     {{< figure-img figure-class="figure-containerf" src="/images/pupil-hardware/eyerotate.gif" >}}
+     {{< figure-img figure-class="figure-container" src="/images/pupil-hardware/eyerotate.gif" >}}
```
Comment (willpatera) [inline pupil-hardware/pupil-headset.md]: Nice fix.

**Comment by nathakits** [comment]
Requested changes are fixed. Ready to merge.

**Pull request merged**